doc/CONFIGURATION.md

```diff
@@ -279,28 +279,94 @@ To increase the maximum object size for writes, use the `--write-part-size` comm
+
+The options field takes regular Mountpoint CLI arguments as comma separated values in the form `key=value`. For example if you previously used `--allow-delete --uid 7` as CLI arguments you would use `allow-delete,uid=7` in your fstab file.
+
+If you need to include commas in your CLI argument, for example `--cache /tmp/foo,bar`, you would escape the commas in the argument with backslashes like so: `cache=/tmp/foo\,bar`. Characters that need backslash escaping include backslashes (`\`), commas (`,`), and double quotes (`"`).
+
+To stop Mountpoint from running at boot, remove the corresponding line from your fstab file. The filesystem will no longer be automatically mounted on subsequent reboots.
+
+#### Validating changes to the fstab file
+
+If your fstab file is invalid, your operating system may fail to boot. We recommend making a backup of your fstab file before changes, and also using the `nofail` option to ensure invalid Mountpoint configuration or startup failures do not prevent the instance from booting.
+To validate changes, you can use the following snippet, changing `/mnt/mountpoint` to the local path specified in your fstab file:
+
+```
+MNT_PATH=/mnt/mountpoint
+sudo systemctl daemon-reload
+sudo systemctl restart "$(systemd-escape --suffix=mount --path $MNT_PATH)"
+sudo systemctl status "$(systemd-escape --suffix=mount --path $MNT_PATH)"
+```
+
+#### Example output for a successful mount:
+
+```
+● mnt-mountpoint.mount - /mnt/mountpoint
+     Loaded: loaded (/etc/fstab; generated)
+     Active: active (mounted) since Tue 2025-05-20 10:53:36 UTC; 16s ago
+      Where: /mnt/mountpoint
+       What: mountpoint-s3
+       Docs: man:fstab(5)
+             man:systemd-fstab-generator(8)
+      Tasks: 17 (limit: 9347)
+     Memory: 22.8M
+        CPU: 72ms
+     CGroup: /system.slice/mnt-mountpoint.mount
+             └─476 /sbin/mount.mount-s3 amzn-s3-demo-bucket /mnt/mountpoint -o rw,nosuid,nodev,allow-other,_netdev
+
+May 20 10:53:35 ip-172-31-40-171 systemd[1]: Mounting mnt-mountpoint.mount - /mnt/mountpoint...
+May 20 10:53:35 ip-172-31-40-171 mount[466]: Using 'fstab' style options as detected use of `-o` argument.
+May 20 10:53:36 ip-172-31-40-171 mount[466]: bucket amzn-s3-demo-bucket is mounted at /mnt/mountpoint
+May 20 10:53:36 ip-172-31-40-171 systemd[1]: Mounted mnt-mountpoint.mount - /mnt/mountpoint.
+```
+
+By validating the fstab file, you can be confident that Mountpoint will be automatically mounted after subsequent reboots.
+
+#### Using EC2 User Data to install and configure Mountpoint
+
+You can setup Mountpoint as part of new instance launches by including a user data script to install Mountpoint and configure the fstab file.
+
+#### Example user data script (tested on AL2023)
```

Comment:
Maybe drop this heading? Keep it as a paragraph instead.

I'd also clarify that this is for x86 instances.